.github/workflows/release.yml

```diff
@@ -0,0 +1,12 @@
+name: Release new version
+on:
+  release:
+    types:
+      - released
+
+jobs:
+  build-java:
+    uses: exacaster/lighter/.github/workflows/build-java.yml@main
```

Comment:
Should we someday extract these to separate repo?

Reply:
Ups, forgot to change it to delta-fetch after copying :D.

I think it is fine to repeat those in both of our opensource projects, to allow us to make pipeline changes independently.